library/Director/ProvidedHook/Icingadb/CustomVarRenderer.php

```diff
@@ -130,6 +135,13 @@ public function renderCustomVarValue(string $key, $value)
 
             if (isset($this->datalistMaps[$key][$value])) {
                 return $this->datalistMaps[$key][$value];
+<<<<<<< Updated upstream
+            } elseif (in_array($key, $this->dictionaryNames)) {
+                return $this->renderDictionaryVal($value);
+=======
+            } elseif ($value !== null && in_array($key, $this->dictionaryNames)) {
+                return $this->renderDictionaryVal($key, (array) $value);
+>>>>>>> Stashed changes
```

Comment:
please amend this, syntax errors shouldn't remain anywhere in commits

Reply:
Both commits before the phpstan baseline update seem to be duplicates of those right after the baseline update. Please merge them.